### ISSUE TITLE
PBI: Example

### ISSUE THREAD

**Issue by MartijnWoudstra** [mindhashnl/roomsignage#2]
This is an example of a PBI (Product Backlog Item). PBI's have tasks which are clear and seperated from other tasks. Dependency can occur. E.G.

PBI: Write questions first interview
- [x] Make up and write down our general idea (with @someone)
- [x] Write down what different possibilities are possible
- [x] Write down questions that would answer these questions
- [ ] Inform rest of group
- [ ] Write down result of the interview

Milestones are Epics. Epics are a collection of PBI's. Epics are big features which require multiple PBI's. One example of an epic could be: Being able to upload company picture and display it.
PBI's are tasks which are defined by the design.
Bugs are bugs introduced by implementing PBI's. Severities are as follows: 
- Mayor: Fix immediately
- Normal: Fix next sprint
- Minor: Fix when currently accepted PBI's are done.

PBI's are accepted by everyone before they enter a sprint. Accepting means a clear goal, workplan and testplan. PBI's are finished when testplans are executed without errors, or when the errors are accepted as bugs.

PBI's with their tasks can be submitted by any teammember. The scrummaster facilitates the discussion and thus prepares the discussion. 

In short: Epics (milestones) are made. PBI's with tasks are submitted by the team, prepared by the scrummaster, and accepted by the team (if criteria are met). PBI's are closed when testplan reveals no bugs, or bugs are accepted as bugs. 

Branch management:
- master is main branch
- development is the branch which is pulled into master after every sprint
- Every PBI gets it's own branch which is PR'd into the dev branch

Join slack: [Slack invitation](https://join.slack.com/t/mindhashproject/shared_invite/enQtNzQ1NDU1NDg2MzI0LTQzNjg0NTBjMDMzMGM3NzVkMTcxODZjNmIwZGRhNzA1ODVkYTU0MjRiY2JkYjY4MzgwMzRhNDAxNTZhZjAyODk)
